src/controls/measure.js

```diff
@@ -65,6 +68,8 @@ const Measure = function Measure({
   let measureButton;
   let lengthToolButton;
   let areaToolButton;
+  let bufferToolButton;
+  let bufferSize = 1000;
```

Comment:
No need to set a default value if the default value is never used.

Reply:
That was a remnant from before reworking code. Good catch.